### PR TITLE
Add UI coverage for Data Docs

### PR DESCRIPTION
**Pull request by megcrenshaw**
Finishes up data docs UI tests by clicking on the create, edit, and destroy buttons on the edit_all page. Creates the data doc on the new page, edits the data doc on the edit page, and verifies we see the created doc on both the index page and on its own page.

Video: 

https://user-images.githubusercontent.com/9142121/195440283-7563829d-2169-4151-ba55-b33749b11678.mp4

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: dashboard/test/ui/features/ed_programs/levelbuilder/create_and_delete_data_docs.feature [hunk 1]
```diff
@@ -0,0 +1,48 @@
+@no_mobile
```
Comment (davidsbailey) [inline dashboard/test/ui/features/ed_programs/levelbuilder/create_and_delete_data_docs.feature]: nit: move this file into `learning_platform/levelbuilder` ?

File: dashboard/test/ui/features/step_definitions/levelbuilder_steps.rb [hunk 2]
```diff
@@ -161,16 +161,26 @@
...
-  )
+Given(/^I click the icon to edit the temp data doc$/) do
+  steps %{
+    And I click selector "#edit_#{@temp_data_doc_key}"
+  }
+end
+
+Given(/^I click the icon to delete the temp data doc$/) do
+  steps %{
+    And I click selector "#delete_#{@temp_data_doc_key}"
```
Comment (davidsbailey) [inline dashboard/test/ui/features/step_definitions/levelbuilder_steps.rb:178]: nice strategy!

**Review by davidsbailey — APPROVED** [review]
nice job!

**Comment by megcrenshaw** [comment]
Good call re: moving this into levelbuilder! That happened as part of the merge, and then I deleted an unused method too

**Pull request merged**